src/migrations/0001.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Cronbeat\Migrations;
+
+use Cronbeat\BaseMigration;
+use Cronbeat\Logger;
+
+/**
+ * Migration 0001: Initial schema setup
+ * Creates the users and migrations tables
+ */
+class Migration0001 extends BaseMigration {
```

Comment:
@jetbrains-junie flag the migration classes so that "PhpUnused" is not inspected.